romancal/regtest/test_tweakreg.py

```diff
@@ -81,6 +81,47 @@ def test_tweakreg(
 
     assert rms < 1.3 / np.sqrt(2)
 
+    # check if the Mean Absolute Error is less that 10 milliarcsec  (DMS406)
+    abs_diff = (np.absolute(diff)) / 1.0e-3
+    mean_abs_error = (tweakreg_out.meta.wcs_fit_results.mae) / 1.0e-3
+    assert mean_abs_error < 10.0
+    assert np.max(abs_diff) < 10.0
+    passmsg = "PASS" if np.max(abs_diff) < 10.0 else "FAIL"
+    dms_logger.info(
+        f"DMS406 {passmsg} the Absolute Astrometric Uncertainty of {np.max(abs_diff):5.2f} mas is less that 10 mas."
+    )
+
+    assert tweakreg_out.meta.cal_step.tweakreg == "COMPLETE"
```

Comment:
```suggestion
```
This is a duplicate assert:
https://github.com/spacetelescope/romancal/blob/21dcb64a7cfd31a1d4a72c30fda5fb551e076fdd/romancal/regtest/test_tweakreg.py#L49